tests/lib/Client/fixtures.ts

```diff
@@ -37,3 +39,21 @@ export const STATE_REQUEST_BODY_WITH_CUSTOM_VARIABLES: RequestContext = {
   },
   request: null,
 };
+
+export const INTERACT_AXIOS_POST_RESPONSE_SPEAK_TRACE: BaseSpeakTrace = {
```

Comment:
I would just make this into `BASE_SPEAK_TRACE`, because this data seems reusable and we should signal to the reader that is a "basic example" of `BaseSpeakTrace` data.